src/Routing/Route/Route.php

```diff
@@ -423,6 +423,7 @@ public function match(array $url, array $context = [])
             $hostOptions['_ext'] = $url['_ext'];
             unset($url['_ext']);
         }
+        unset($defaults['_ext']);
```

Comment:
I should also mention @markstory that I was just like you, thinking that `resources()` would create routes just like `fallbacks()` does without breaking anything compared to manually creating them using `connect()` - but it turns out it's a little more complex than `fallbacks()`.